functional/sev/run.sh

```diff
@@ -0,0 +1,337 @@
+  sleep 20
+
+  # Log kubectl environment information: nodes, services, deployments, pods
+  # Retrieve pod name and IP
+  esudo kubectl get nodes -o wide
+  echo "-------------------------------------------------------------------------------"
+  esudo kubectl get services -o wide
+  echo "-------------------------------------------------------------------------------"
+  esudo kubectl get deployments -o wide
+  echo "-------------------------------------------------------------------------------"
+  esudo kubectl get pods -o wide
+  echo "-------------------------------------------------------------------------------"
+  pod_name=$(esudo kubectl get pod -o wide | grep ccv0 | awk '{print $1;}')
+  esudo kubectl describe pod ${pod_name}
+  echo "-------------------------------------------------------------------------------"
+  pod_ip=$(esudo kubectl get pod -o wide | grep ccv0 | awk '{print $6;}')
+  
+  # Look for SEV enabled in container dmesg output
+  sev_enabled=$(ssh -i test/ccv0-ssh -o "StrictHostKeyChecking no" -t root@${pod_ip} 'dmesg | grep SEV' || true)
+  if [ -z "$sev_enabled" ]; then
+    >&2 echo -e "${RED}KATA CC SEV TEST - FAIL: SEV is NOT Enabled${NC}"
+    echo "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@"
+    exit 1
+  else
+    echo "DMESG REPORT: $sev_enabled"
+    echo -e "${GREEN}KATA CC SEV TEST - PASS: SEV is Enabled${NC}"
+    echo "@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@"
+  fi
+  popd
+}
+
+cleanup() {
+  exit_code=$?
+  set +eE; set +o nounset +o pipefail
+
+  if [ ${exit_code} -ne 0 ]; then
+    >&2 echo -e "${RED}???????????????????????????????????????????????????????????????????????????????${NC}"
+    >&2 echo -e "${RED}ERROR Encountered with Exit Code: ${exit_code}${NC}"
+    >&2 echo -e "${RED}???????????????????????????????????????????????????????????????????????????????${NC}"
+  fi
+
+  echo "###############################################################################"
+  echo "CLEANUP - STARTED"
+
+  # Remove the service/deployment/pod and uninstall kubernetes
+  esudo kubectl delete -f test/ccv0-ssh.yaml 2>/dev/null
+  esudo "${tests_repo_dir}/integration/kubernetes/cleanup_env.sh"
+
+  # Stop KBS DB container and prune it
+  esudo docker stop kbs-db 2>/dev/null
+  esudo docker container prune -f 2>/dev/null
+
+  # Kill KBS and attestation-agent daemons
+  esudo pkill -9 -f simple-kbs
+  esudo pkill -9 -f sample_keyprovider
+  sleep 5
+
+  echo "Cleanup complete"
+  echo "###############################################################################"
+  exit $exit_code
+}
+
+main() {
+  mkdir -p test
+  source ~/.profile
+
+  # Verify docker authentication
+  if ! esudo cat /root/.docker/config.json | grep auths >/dev/null; then >&2 echo "ERROR: Not logged into docker"; exit 1; fi
+  esudo docker login
+
+  # kata config and kernel params
+  modify_kata_config
```

Comment:
Will we still need this once #5013 is merged?

Reply:
Nope. I'll remove it when 5013 is in.